python/feathub/feature_views/feature_view.py

```diff
@@ -169,3 +170,18 @@ def _get_keys(self) -> Optional[List[str]]:
                 key_fields.extend(keys)
 
         return list(OrderedDict.fromkeys(key_fields))
+
+    def is_bounded(self) -> bool:
+        return cast(TableDescriptor, self.source).is_bounded()
```

Comment:
Should we re-use `get_resolved_source()` here so that it throws exception if the source is unresolved?